passport_test.go

```diff
@@ -382,7 +386,9 @@ func TestVerifyProof(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			err = verifier.VerifyProof(validProof, tc.verifyOpts...)
+			//err = verifier.VerifyProof(validProof, tc.verifyOpts...)
```

Comment:
commented code reason?

Reply:
Was testing and forgot to remove comments